app/assets/stylesheets/helpers/_text.scss

```diff
@@ -190,7 +190,7 @@ article {
   }
 
   th, td {
-    @include core-16($tabular-numbers: true);
```

Comment:
What did `tabular-numbers` do, and is it no longer needed?

Reply:
I must have lost it in the process. 🤦🏻‍♀️ 
I believe what it does is request the tabular version of the Transport font.

Have updated to pass an additional `$tabular: true` param to the `govuk-font` mixin (as per [here](https://github.com/alphagov/govuk-frontend/blob/9dc2cfb064fb49e421d5a73be6a241506e1414b8/src/govuk/helpers/_typography.scss#L177))